components/eam/src/physics/cam/clubb_intr.F90

```diff
@@ -3173,14 +3248,44 @@ subroutine clubb_surface (state, cam_in, ustar, obklen)
          thv(i) = th(i)*(1._r8+zvir*state%q(i,pver,ixq))  ! diagnose virtual potential temperature
        end if
     enddo
-
+    !
     do i = 1, ncol
        call calc_ustar( state%t(i,pver), state%pmid(i,pver), cam_in%wsx(i), cam_in%wsy(i), &
                         rrho, ustar(i) )
        call calc_obklen( th(i), thv(i), cam_in%cflx(i,1), cam_in%shf(i), rrho, ustar(i), &
                         kinheat, kinwat, kbfs, obklen(i) )
     enddo
-
+    !
+    if (use_od_ss) then
+    !add calculation of bulk richardson number here
+    !
+    !compute the whole level th and thv for diagnose of bulk richardson number
+    thv_lv=0.0_r8
+    th_lv=0.0_r8
+    !
+    do i=1,ncol
+      do k=1,pver
+         th_lv(i,k) = state%t(i,k)*state%exner(i,k)
+         if (use_sgv) then
+           thv_lv(i,k) = th_lv(i,k)*(1.0_r8+zvir*state%q(i,k,ixq) &
+                    - state%q(i,k,ixcldliq))  !PMA corrects thv formula
```

Comment:
is "PMA" for someone's initials? We don't want tags like this in the code.

Reply:
Also - a better description about what is being fixed would be better since it's very unclear what the problem was.